pgmpy/base/DirectedGraph.py

```diff
@@ -177,12 +171,11 @@ def moralize(self):
 
         Examples
         --------
-        >>> from pgmpy.base import DirectedGraph
         >>> G = DirectedGraph([('diff', 'grade'), ('intel', 'grade')])
         >>> moral_graph = G.moralize()
-        >>> moral_graph.edges()
+        >>> moral_graph.edges() # doctest: +IGNORE_RESULT
```

Comment:
@nishnik Did you went through this link https://docs.python.org/2/library/doctest.html#unittest-api , I believe python provides a good doctest API, I'll also talk to someone at sympy and see how do they deal with doctests, will let you know on this thread. Till then you should go through the mentioned link.